src/game_api/entity.cpp

```diff
@@ -244,22 +244,14 @@ std::pair<float, float> Entity::position_self() const
 
 std::pair<float, float> Entity::position_render() const
 {
-    if (overlay != nullptr)
-    {
-        auto [x_pos, y_pos] = position_self();
-        auto [rx_pos, ry_pos] = overlay->position_render();
-        return {rx_pos + x_pos, ry_pos + y_pos};
-    }
-    if (rendering_info == nullptr)

```

Comment:
Why drop this verification?

Reply:
well you could use that argument for any code, like we have tons of functions that access stuff in state. It's a little over protective to check if every single pointer